gateway/tests/test_scim.py

```diff
@@ -2,6 +2,7 @@
 # All rights reserved.
 # See LICENSE file in this distribution.
 # SPDX-License-Identifier: Apache-2.0
+# pylint: disable=consider-using-with
```

Comment:
Why is this required?

Reply:
See above

Reply:
Take it out and see what happens ;-)

Reply:
Oh its because of 
```
voucher = open("tests/fdo-voucher.b64", encoding="utf-8").read()
```
Why didn't you use `with` instead of disabling it here?

Reply:
Because the error is spurious.  This does the job in one line instead of two, and it's a test, so there is no need to trap.